src/eth_provider/utils.rs

```diff
@@ -88,4 +90,28 @@ mod tests {
             doc! {"test_key": format!("0x{}", "0".repeat(64))}
         );
     }
+
+    #[test]
+    fn test_split_u256() {
+        // Define a property to test the split_u256 function
+        proptest!(|(value in any::<B256>())| {
```

Comment:
we can't do `any::<U256>`?

Reply:
Yes nice observation, didn't notice this was implemented for `U256`. The test looks much better like that.